seatunnel-examples/seatunnel-flink-connector-v2-example/src/main/resources/examples/fake_to_kafka.conf

```diff
@@ -0,0 +1,34 @@
+env {
```

Comment:
Please revert this file. In fact we already ignore the file add in `seatunnel-examples` module from `.gitignore` file. I don't know why it seems not work in your pull request.